datafusion/sqllogictest/test_files/array.slt

```diff
@@ -689,7 +689,7 @@ select column1, column2, column3, column4, column5 from nested_arrays;
 
 # values table
 query IIIRT
-select a, b, c, d, e from values;
+select a, b, c, d, e from "values";
```

Comment:
it seems that `values` can no longer be parsed as a bare table name (I am not sure why, to be honest)

Reply:
I guess it's because we remove the "UnQuotedString" and values is not a word?

Reply:
It's because of [this PR](https://github.com/sqlparser-rs/sqlparser-rs/pull/1249), which made `values` . I'll make a fix.

Reply:
https://github.com/sqlparser-rs/sqlparser-rs/pull/1278

Reply:
Push a new commit to revert this work around. It's now fixed.